docker/pytorch-aarch64/Dockerfile

```diff
@@ -290,13 +290,75 @@ ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 COPY scripts/build-pytorch.sh $PACKAGE_DIR/.
 COPY patches/onednn.patch $PACKAGE_DIR/.
 RUN $PACKAGE_DIR/build-pytorch.sh
+
+# Install torchvision, torchdata and torchtext
+RUN pip install --no-cache-dir torchvision==$TORCHVISION_VERSION \
+    torchdata==$TORCHDATA_VERSION
+ENV VENV_PACKAGE_DIR=$VENV_DIR/lib/python$PY_VERSION/site-packages
+COPY scripts/build-torchtext.sh $PACKAGE_DIR/.
+RUN $PACKAGE_DIR/build-torchtext.sh
+
+CMD ["bash", "-l"]
+# ========
+# Stage 4b: build PyTorch-xla
+# ========
+FROM pytorch-libs AS pytorch-xla
+ARG njobs
+ARG onednn_opt
+ARG default_py_version
+ARG cpu
+
+ENV ONEDNN_BUILD="${onednn_opt}" \
+    NP_MAKE="${njobs}" \
+    CPU="${cpu}"
+
+# Key version numbers
+ENV PY_VERSION="${default_py_version}" \
+    ONEDNN_VERSION="v2.6" \
+    TORCH_VERSION=1.11.0 \
+    TORCHVISION_VERSION=0.12.0 \
+    TORCHDATA_VERSION=0.3.0 \
+    TORCHTEXT_VERSION=0.12.0
+
+# Use a PACKAGE_DIR in userspace
+WORKDIR /home/$DOCKER_USER
+USER $DOCKER_USER
+ENV PACKAGE_DIR=/home/$DOCKER_USER/$PACKAGE_DIR
+RUN mkdir -p $PACKAGE_DIR
+
+# Copy in the Python virtual environment
+ENV VIRTUAL_ENV=/home/$DOCKER_USER/python3-venv
+COPY --chown=$DOCKER_USER:$DOCKER_USER --from=pytorch-tools $VIRTUAL_ENV /home/$DOCKER_USER/python3-venv
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+# Get Bazel binary for AArch64 using the latest Bazelisk release
+RUN mkdir -p $PACKAGE_DIR/bazel && \
```

Comment:
I tried bazelisk (from tensorflow) but it didn't work here, I need to check if any version issue.

Reply:
What was the problem? Presumably getting the latest bazelisk, with this command was ok? Was bazelisk then unable to choose the right Bazel version for the build? Since we're not running `bazel` from within TensorFlow, I could easily believe that bazelisk is unable to ascertain the right version, as I think it may need `bazel` to be called from within `tensorflow/` to do this...

So, if we have to use specific Bazel version, since the Bazel version number appears twice in the URL, could you replace it with a single variable, set at the start of the Docker stage (or even in build.sh) as we used to for the TF build - https://github.com/ARM-software/Tool-Solutions/blob/574c8e96c993818a8600fe6a50ca34586b958041/docker/tensorflow-aarch64/Dockerfile#L275
...you could just lift that `get-bazel.sh` from the older TF builds perhaps?